package.json

```diff
@@ -38,29 +38,26 @@
     "prepack": "npm run build"
   },
   "devDependencies": {
-    "@balena/lint": "^6.1.1",
-    "@types/chai": "^4.2.19",
-    "@types/chai-as-promised": "^7.1.4",
-    "@types/dockerode": "^2.5.34",
-    "@types/mocha": "^8.2.2",
-    "@types/node": "^14.17.4",
-    "chai": "^4.3.4",
-    "chai-as-promised": "^7.1.1",
-    "dockerode": "^3.3.0",
```

Comment:
@Page- Was `dockerode` and `node-docker-api` unnecessary and just not removed before?

Reply:
Yup, that's why they're under the "removed unused deps" commit